tests/layer_tests/onnx_tests/test_loop.py

```diff
@@ -0,0 +1,401 @@
+# Copyright (C) 2018-2021 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+import numpy as np
+import pytest
+
+from common.onnx_layer_test_class import OnnxRuntimeLayerTest
+
+
+class TestLoop(OnnxRuntimeLayerTest):
+
+    def create_loop(self):
+        """
+            ONNX net
+
+            Input->Loop->Output   =>   Only accuracy check
+
+        """
+
+        #   Create ONNX model
+
+        import onnx
+        from onnx import helper
+        from onnx import TensorProto
+
+        input_shape = [1, 4, 64, 54]
+
+        in_1 = helper.make_tensor_value_info('IN_1', TensorProto.FLOAT, input_shape)
+        in_1_int = helper.make_tensor_value_info('in_1_int', TensorProto.FLOAT, input_shape)
+        in_1_int_out = helper.make_tensor_value_info('in_1_int_out', TensorProto.FLOAT, input_shape)
+
+        out_1 = helper.make_tensor_value_info('OUT_1', TensorProto.FLOAT, None)
+        res = helper.make_tensor_value_info('res', TensorProto.FLOAT, None)
+
+        m_1 = helper.make_tensor_value_info('m_1', TensorProto.INT64, [1])
+
+        cond_int_1 = helper.make_tensor_value_info('cond_int_1', TensorProto.BOOL, [1])
+        cond_out_1 = helper.make_tensor_value_info('cond_out_1', TensorProto.BOOL, [1])
+
+        m_1_value = np.array([10], dtype=np.int64)
+        cond_value = np.array([True], np.bool)
+        one_value = np.ones(input_shape, dtype=np.float)
+
+        M_1 = onnx.helper.make_node(
+            'Constant',
+            inputs=[],
+            outputs=['M_1'],
+            value=onnx.helper.make_tensor(
+                name='const_tensor',
+                data_type=onnx.TensorProto.INT64,
+                dims=m_1_value.shape,
+                vals=m_1_value.flatten().astype(np.int64),
+            ),
+        )
```

Comment:
we have a lot of duplications here for creating several constants. We can create a function for creating a constant node for specified type and name. It will greatly eliminate code here and readability.